### PR TITLE
perf: Removal of non-necessary includes in the engine

### PR DESCRIPTION
**Pull request by sncarbo**

### CODE REVIEW

File: Source/Modules/ModuleEditor.cpp [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
 #include "Windows/WindowMainMenu.h"
 #include "Windows/EditorWindows/WindowConsole.h"
 #include "Windows/EditorWindows/WindowScene.h"
 #include "Windows/EditorWindows/WindowConfiguration.h"
 #include "Windows/EditorWindows/WindowInspector.h"
 #include "Windows/EditorWindows/WindowHierarchy.h"
-#include "Windows/EditorWindows/WindowFileBrowser.h"
 #include "Windows/EditorWindows/WindowEditorControl.h"
 
-#include <ImGui/imgui.h>
```
Comment (OriolSors) [inline Source/Modules/ModuleEditor.cpp:18]: Hi @sncarbo ! It seems that there is a conflict here:
Captura de pantalla 2023-02-24 a las 14 29 19
Can you check it?

**Review by OriolSors — COMMENTED** [review]
Despite the conflict with ModuleEditor, the rest has been cleaned! So good job @sncarbo !

**Comment by sncarbo** [comment]
@OriolSors Thank you very much for your nice comment Oriol!

**Comment by sncarbo** [comment]
The conflicts have been solved!

**Review by gerard-queralt — APPROVED** [review]
Thank you for your work Santi!

**Comment by sncarbo** [comment]
@gerard-queralt Thank you Gerard! I appreciate it.

**Comment by sncarbo** [comment]
@OriolSors I'm sorry to bother you again, would you mind checking the pull request again? I don't think there should be any conflict anymore!

**Review by OriolSors — APPROVED** [review]
Fine @sncarbo! Good job again and the conflicts are solved!

**Comment by sncarbo** [comment]
@OriolSors Thank you very much Oriol!

**Pull request merged**